release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2643,3 +2643,16 @@ link:https://access.redhat.com/solutions/6324771[{product-title} 4.8.11 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+// TODO: Add this to the end of the appropriate 4.8 z-stream RNs that this check is added with.
+[id="ocp-4-8-preparing-upgrade-next-release"]
+==== Preparing to upgrade to the next {product-title} release
+
+// TODO: replace "TODO" with appropriate z-stream, and is "check" the correct term?
```

Comment:
Now that I've read this and the kbase article a couple of times, I realized that the following sentence: 

"...introduced a check that will impact upgrading to the next {product-title} release..."

could change to:

"...introduced a check that impacts upgrading to the next {product-title} release..."

I don't think this change would impact the language in the kbase article, but could help in updating this RN post-GA.